neal/sampler.py

```diff
@@ -208,11 +233,12 @@ def sample(self, bqm, beta_range=None, num_reads=None, num_sweeps=1000,
         # parse_initial_states could handle seed generation, but because we're
         # sharing it with the SA algo, we handle it out here
         if seed is None:
-            seed = randint(0, (1 << 32 - 1))
+            seed = randint(2**32)
```

Comment:
"seed = randint(2**32)" --> the `randint()` function requires two input arguments. I think you likely wanted "seed = randint(0, 2**32)". See https://docs.python.org/3/library/random.html#random.randint

Reply:
https://numpy.org/doc/stable/reference/random/generated/numpy.random.randint.html
Documentation seems to be inconsistent across various sites on this. Lower bound is an optional argument. I've tested this and it works fine.

Reply:
Main aim here is to replace the confusing (1 << 32 - 1): this is confusing because it is incompatible with the range bounds reported and used in subsequent statements, and hard to remember 'order of precedence' for subtraction and bit shift. Adding back the 0 argument is easy enough.

Reply:
The file has
```
from random import randint
```
Not the function from NumPy

Reply:
You got me

Reply:
I think I am going to import randint from numpy.random instead of random. This would be more consistent with the use of numpy.random as the standard PRNG elsewhere in the code (and elsewhere in ocean). The bounds used in this part of the code are also specific to seeding of numpy.random.RandomState, and are confusing in the context of random.seed() which accepts values outside these bounds.

Reply:
Can it be clarified if the current implementation with np.random.randint() fails (aside from deprecation issue).